raiden/api/python.py

```diff
@@ -147,19 +147,27 @@ def leave_token_network(self, token_address, only_receiving=True):
         return connection_manager.leave(only_receiving)
 
     def get_connection_manager_funds(self, token_address):
-        """Get the connection manager for a specific token"""
-        connection_manager = self.raiden.connection_manager_for_token(token_address)
-        if not connection_manager:
-            return None
+        """Get the sum of all connection manager's open channels deposits"""
```

Comment:
Now the docstring has correctly changed, but the method name is misleading: the connection manager's `funds` attribute is part of the connection strategy. This method should now be called `get_connection_manager_deposit_sum`.

Reply:
Ok, will change.